Makefile

```
@@ -8,27 +8,53 @@ GOARCH_aarch64 = arm64
 GOARCH_arm64 = arm64
 GOARCH ?= $(shell echo "$(GOARCH_$(ARCH))")
 
+VERSION := $(shell git describe --tags --always)
+REVISION := $(shell git rev-parse HEAD)
+PACKAGE := github.com/abiosoft/colima/config
+VERSION_VARIABLES := -X $(PACKAGE).appVersion=$(VERSION) -X $(PACKAGE).revision=$(REVISION)
+
+OUTPUT_DIR := _output/binaries
+OUTPUT_BIN := colima-$(OS)-$(ARCH)
+INSTALL_DIR := /usr/local/bin
+BIN_NAME := colima
+
+LDFLAGS := $(VERSION_VARIABLES)
+
+.PHONY: all
 all: build 
 
+.PHONY: clean
 clean:
 	rm -rf _output _build
 
+.PHONY: gopath
 gopath:
 	go get -v ./cmd/colima
 
+.PHONY: fmt
 fmt:
 	go fmt ./...
 	goimports -w .
 
+.PHONY: build
 build:
-	GOOS=$(GOOS) GOARCH=$(GOARCH) OS=$(OS) ARCH=$(ARCH) sh scripts/build.sh
+	go build -ldflags="$(LDFLAGS)" -o $(OUTPUT_DIR)/$(OUTPUT_BIN) ./cmd/colima
+	openssl sha256 -r -out $(OUTPUT_DIR)/$(OUTPUT_BIN).sha256sum $(OUTPUT_DIR)/$(OUTPUT_BIN)
```

I am hoping the hash file can stay consistent as well.

It is now changed from the following format
```
3fc57d9ca92c802a4a4b0745ffe782e3a1074898530da48802f0097ab36bff58  colima-Darwin-x86_64
```

to the following format
```
9619fa6137350d4060127cd4e241f74f87bda0663bb1077e08e9f62c50ee1579 *_output/binaries/colima-Darwin-x86_64
```

I tried the following but could not get rid of the preceding asterisk `*`.

```suggestion
	cd $(OUTPUT_DIR) && openssl sha256 -r -out $(OUTPUT_BIN).sha256sum $(OUTPUT_BIN)
```

---

@abiosoft
The preceding asterisk has a meaning: it means that the input file is [binary](https://apple.stackexchange.com/questions/230917/need-a-cli-to-check-the-sha256-hash-of-a-file).

Openssl automatically recognises that the colima bin is in binary format whereas shasum does not because it read in text mode by default. So if you want to get rid of the asterisk we need to get back to using `shasum`/`sha256sum` depending on the OS.

---

If it's a known standard then I am fine with it. 

Thanks.